app/index.js

```diff
@@ -21,7 +20,7 @@ const init = async () => {
   console.log('Server running on %s', server.info.uri)
 }
 
-process.on('unhandledRejection', err => {
+process.on('unhandledRejection', (err) => {
   console.log(err)
   process.exit(1)
 })
```

Comment:
kill server on error?

Reply:
Think this is just a formatting change on my part — have removed the parenthesis.